PB-558: Overriding Page Builder Styles using themes (#16)
magento/magento2-page-builder-docs: docs/styling/override-pagebuilder-styles-themes.md

4. **_extend.less**. The `_extend.less` file is required and must be added directly to your frontend theme's `source` directory. Magento uses this file to add your frontend styles to the `pub/static/frontend` output in a location within the `styles-m.css` where they override (instead of replace) the default frontend styles, including Page Builder's default content-type styles. Like the `_import.less` files, the `_extend.less` file should only contain `@import` statements. In our example, our `_extend.less` contains the same two imports as seen in `module.less`:

    ```scss
    @import 'pagebuilder/heading/_import.less';
    @import 'pagebuilder/products/_import.less';
    ```

### Step 2: Apply the frontend theme

The following instructions are here as a **reminder** to apply your frontend theme to a store view or page. If you forget, you won't see your style overrides in the storefront, and you will wonder why. To learn more on applying themes in general, see [[Apply a storefront theme]({{ site.baseurl }}/guides/v2.4/frontend-dev-guide/themes/theme-apply.html).

**To apply your frontend theme to a store view**:

> Navigate to Content > Design > Configuration and edit the store view where you can apply your frontend theme as the default theme:

Author: Maybe? Here's what I see. 
![image](https://user-images.githubusercontent.com/1828494/102406717-9dd97400-3fb0-11eb-8b36-2ce10f555f16.png)

![Set default theme on store view](../images/theme-default-setting-admin.svg)
_Set default frontend theme for store view_

**To apply your frontend theme to a page**:

> During development, it's easy to apply your frontend theme to a single page for testing. But equally easy to forget to switch your page to that theme. Consider this your reminder.

Reviewer: ```suggestion
During development, it's easy to apply your frontend theme to a single page for testing, but it's also easy to forget to switch your page to that theme. 
```

The "Consider" sentence was too harsh.
